src/lib/provable/bytes.ts

```diff
@@ -44,7 +44,7 @@ class Bytes {
    *
    * Inputs smaller than `this.size` are padded with zero bytes.
    */
-  static from(data: (UInt8 | bigint | number)[] | Uint8Array | Bytes): Bytes {
+  static from(data: FlexibleBytes): Bytes {
```

Comment:
FYI, I often avoided putting types in method signatures that are not exported, because for users the DX is better when they see an explicit type `(UInt8 | bigint | number)[] | Uint8Array | Bytes` instead of an opaque one.